### ISSUE TITLE
Parental controls blocking and not allowing domains despite having custom filters

### ISSUE THREAD

**Issue by caressofsteel** [AdguardTeam/AdGuardHome#5723]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

Other (please mention in the description)

### CPU architecture

ARMv7

### Installation

Other (please mention in the description)

### Setup

On a router, DHCP is handled by the router

### AdGuard Home version

0.107.6

### Description

I cannot visit a specific domain without having Parental Controls OFF. The domain (etsy.com) is not a blocked website in any of my rules or filters, and to my knowledge not in any block lists for adult content. Adding custom filters to allow the domain have no effect, I am forced to turn OFF parental controls before the site is able to be visited. 

If I leave parental controls on and try to visit the site with custom allow filters, the site will return HSTS and SSL errors. If I turn off parental controls and hard refresh the browser the site will load normally.

Here is the report on the site in question:
https://reports.adguard.com/en/www.etsy.com/report.html?aid=16593&utm_campaign=site_report_page&utm_source=adguard_dnsparental

I run AGH on a Nighthawk 7800 router alongside OpenWRT. It's been a going strong for a few years now.

**Comment by JuanRodenas** [comment]
Hello!

It has happened to me with Etsy and also with HBO,
here is the report:
https://reports.adguard.com/en/hbo.com/report.html

Greetings,

**Comment by Shiwigy** [comment]
I can confirm this issue with AdGuard parental control web service enabled, even when i mark it as Unblock and verify i have entries in Custom filtering rules according to:
@@||www.hbo.com^$important

It is still being blocked.

It also is reported here https://github.com/AdguardTeam/AdguardFilters/issues/148243#issue-1667727197 that HBO is being blocked by the Parental Control Service.

**Comment by krystian3w** [comment]
It is probably possible to find excerpts from films with physical violence on the site.

**Comment by Shiwigy** [comment]
> It is probably possible to find excerpts from films with physical violence on the site.

Absolutely, but that has nothing to do with the problem. The problem is that we cannot unblock sites that are blocked by the AdGuard Parental Control Web Service.

Off topic:
As you say, there are definitely films with physical violence on HBO.. But Netflix, Disney+ etc. streaming services are not blocked by AdGuard Parental Control Web Service. And they shouldn't either! There is Parental Control built into these services already to block children from watching movies with e.g. physical violence.

**Comment by Shiwigy** [comment]
> Just whether it's effective, falls away as it's a one-click "I'm 18+ years old", falls away as the child knows the pin to the parent/guardian's profile.

You are still off-topic.

If you reason like that, it is just as possible that the children have access to the password for the AdGuard Home server and can log in there. After all, they can even have physical access to the server as there are probably several people running it in their home environment.

The problem remains, we as administrators of our AdGuard Home Servers cannot create rules that allow us to unblock and bypass the AdGuard Parental Control Web Service.

**Comment by krystian3w** [comment]
You can check if the simplified syntax will work:

```adb
@@||www.hbo.com^
@@||hbo.com^
@@||www.etsy.com^
@@||etsy.com^
```

then enough is update tutorial/(knowledge) quide/manual/walkthrough.

Which may indicate that there is no support for force admitting yet.

**Comment by Shiwigy** [comment]
> then enough is update tutorial/(knowledge) quide/manual/walkthrough.
> 
> Which may indicate that there is no support for force admitting yet.

Thank you, i can confirm that the simplified syntax works and allows me to bypass the AdGuard Parental Control Web Service.

With custom filtering rules set to
@@||hbo.com^

13:09:20
2023-04-14
oauth.api.hbo.com
Type: AAAA, Plain DNS
Allowed
Custom filtering rules

So the problem seems to be the ^$important that the Unblock button in the Query Log writes to custom filtering rules

With customer filtering rules set to
@@||hbo.com^$important

13:07:01
2023-04-14
oauth.api.hbo.com
Type: HTTPS, Plain DNS
Blocked by Parental Control
0.10 ms

**Comment by krystian3w** [comment]
About reject unlock etsy: https://github.com/AdguardTeam/AdguardFilters/issues/148111#issuecomment-1506682961

**Comment by caressofsteel** [comment]
I've had these in my custom list for a while. If I remove them I can still get to etsy.com. 

@@||hbo.com^$important
@@||www.hbo.com^$important
@@||etsy.com^$important
@@||www.etsy.com^$important

**Comment by subhappy** [comment]
I'm having this same issue with imgix.

Blocked by parental control:
@@||imgix.net^$important

Not blocked:
@@||imgix.net^

So that's a workaround, but how do I allow subdomains?
In my understanding subdomains should also be whitelisted, but they're still blocked by parental control.
